File: Leetcode/1304 - Find N Unique Integers Sum up to Zero/solution.ts
function sumZero(n: number): number[] {
    const res: number[] = [];
    for (let i = 1; i <= Math.floor(n / 2); i++) {
        res.push(i, -i);
    }
    if (n % 2 === 1) {
        res.push(0);
    }
    return res;
};

File: Leetcode/1304 - Find N Unique Integers Sum up to Zero/solution.py
def sumZero(n: int) -> List[int]:
    res: List[int] = []
    for i in range(1, n // 2 + 1):
        res.append(i)
        res.append(-i)
    if n % 2 == 1:
        res.append(0)
    return res